product/tests/test_services/test_item_creator.py

```
@@ -0,0 +1,159 @@
+            create_item_with_banners(data, self.seller_user)
+
+    def test_atomicity_on_failure(self):
+        # Arrange
+        invalid_data = self.item_data.copy()
+        invalid_data["banners"][0]["image_id"] = 99999
+
+        # Act
+        try:
+            create_item_with_banners(invalid_data, self.seller_user)
+        except ObjectDoesNotExist:
+            pass
```

Ruff (0.8.2)

Use `contextlib.suppress(ObjectDoesNotExist)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(ObjectDoesNotExist)`

(SIM105)